nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeRecord.java

```diff
@@ -451,7 +451,7 @@ public void testMaxSize() {
     }
 
     @Test
-    @Ignore("This unit test depends on timing and could potentially cause problems in an automated build environment. However, it can be useful for manual testing")
+    @Disabled("This unit test depends on timing and could potentially cause problems in an automated build environment. However, it can be useful for manual testing")
```

Comment:
This should be changed to use `EnabledIfSystemProperty`

Reply:
Using nifi.test.performance as the system property to trigger the test